eslint.config.mjs

```diff
@@ -70,8 +89,47 @@ export default tseslint.config(
             'no-iterator': 'error',
             'no-lone-blocks': 'error',
             'no-proto': 'error',
-            'no-new-require': 'error',
```

Comment:
Astute observers may note that this rule has been removed because it's [deprecated](https://eslint.org/docs/latest/rules/no-new-require). It's a pattern we're unlikely to use in modern code, so it's not really worth switching to the replacement.